src/LayerPlan.h

```diff
@@ -380,14 +381,14 @@ class LayerPlan : public NoCopy
      */
     void setPrimeTowerIsPlanned(unsigned int extruder_nr);
 
-    bool getSkirtBrimIsPlanned(unsigned int extruder_nr) const
+    bool getSkirtBrimIsPlanned(const bool for_support, const unsigned int extruder_nr) const
     {
-        return skirt_brim_is_processed[extruder_nr];
+        return (for_support ? support_brim_is_processed : skirt_brim_is_processed)[extruder_nr];
     }
 
-    void setSkirtBrimIsPlanned(unsigned int extruder_nr)
+    void setSkirtBrimIsPlanned(const bool for_support, const unsigned int extruder_nr)
```

Comment:
Same as above, I'd use
```
bool setSkirtBrimIsPlanned(unsigned int extruder_nr, bool for_support = false)
```

Reply:
Done in https://github.com/Ultimaker/CuraEngine/pull/1024/commits/5748ac125f8e025874de29e41c4fab96a18e89ff